feat(elasticsearch): add create index doc (#144)
CleverCloud/documentation: content/doc/addons/elastic/_index.md

{{< callout type="warning" >}}
If you are using Elasticsearch 6, backups are not deleted automatically, you will need to clean them up from time to time.
{{< /callout >}}

## Create an index on Elasticsearch

When creating an Elasticsearch add-on, a single node is setup. By default we apply `cc_singlenode_template` that forces the amount of data replication to zero. This index template is only available for data streams.

> If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is needed. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:

Reviewer: ```suggestion
If you use a tool that requires native Elasticsearch index or that adds a new one through the API, a custom template is needed. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```

Reviewer 2: Can be simply `adds` (add was if the verb was refering to an action from the user, adds if it refers to the tool)

Reviewer 3: How should the final sentence be?

Reviewer 4: @davlgd can you please add your proposition to the content and commit it?

> If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is required. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:

Reviewer 2: ```suggestion
If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is needed. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```

To avoid repetition 🤓

> If you use a tool that requires native Elasticsearch index or that adds a new one through the API, a custom template is needed. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:

Reviewer 2: ```suggestion
If you are using a tool that requires native ES indexes or are accessing them through the ES API, you will need an index template. It can be created either using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```

{{< callout type="warning" >}}
If you are using Elasticsearch 6, backups are not deleted automatically, you will need to clean them up from time to time.
{{< /callout >}}

> ## Create index on Elasticsearch

Reviewer 2: ```suggestion
## Create an index on Elasticsearch
```

> When creating an Elasticsearch add-on, a single node is set-up. Be default we apply an index template `cc_singlenode_template` that force the amount of data replication to zero. This template is only available for data streams.
> 
> To add a new index by API, or when using a tool that requires native Elasticsearch Index, a custom index template is required. This template can be created or by using Kibana or by using the Elasticsearch API.
> 
> To do it with the Elasticsearch API, exec in cURL: 

Reviewer 2: ```suggestion
When creating an Elasticsearch add-on, a single node is setup. By default we apply `cc_singlenode_template` that forces the amount of data replication to zero. This index template is only available for data streams.

If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is required. It can be created using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```

```sh
curl 'https://${ELASTIC_SEARCH_HOST}/_index_template/test/' -u ${ELASTIC_SEARCH_USER} -H 'Content-Type: application/json' -X PUT -d'
{
  "index_patterns" : ["${ELASTIC_INDEX_PATTERN}*"],
  "priority": 2,
  "template": {
    "settings" : {
        "number_of_shards" : 1,
        "number_of_replicas": 0
    }
  }
}
'
```

> This is important here to set a `number_of_replicas` to zero to avoid triggering cluster issues. Priority must be above 1 as `cc_singlenode_template` template has a default one of 1.

Reviewer 2: ```suggestion
It's important here to set `number_of_replicas` to zero to avoid triggering cluster problems. The priority must be greater than 1, as the `cc_singlenode_template` template has a default priority of 1.
```

> This is important here to set a `number_of_replicas` to zero to avoid triggering cluster issues. Priority must be above 1 as `cc_singlenode_template` template has a default one of 1.

Reviewer: ```suggestion
It is important to set a `number_of_replicas` to zero to prevent Elastic Search from creating imaginary cluster issues. Priority must be above 1 as the priority given to `cc_singlenode_template` has a default of 1.
```
